lib/login_register/login.dart

```diff
@@ -190,7 +190,7 @@ class _LoginScreenState extends State<LoginScreen> {
                           FirebaseFirestore.instance
                               .collection('usersCollection')
                               .add(dataToSave);
-                          if (user.emailVerified ?? false) {
+                          if (user.emailVerified) {
```

Comment:
I saw a static analysis warning here, since apparently the value is non-nullable now.